Scrapers/Huone.py

```diff
@@ -0,0 +1,96 @@
+from bs4 import BeautifulSoup
+from requests import get
+import VenueClass
+import json
+from DataFormatting.FormatTime import FormatTime
+from DataFormatting.FormatPrice import FormatPrice
+from DataFormatting.FormatText import FormatTitle
+
+allVenues = []  # Array to store all Venues of type VenueClass
+
+def extractVenue(tag, venue):
+
+    # extract link into the venue
+    venue_link = venue.find_all('a')[0]['href']
+    response = get(venue_link)
+    indiv_venue = BeautifulSoup(response.text, 'html.parser')
+
+    # extract title
+    title_html = indiv_venue.find('div', class_='space-title').find('h1')
+    title = ''
+    if title_html:
+        title = FormatTitle(title_html.text.strip())
+
+    # location - fixed
+    location = 'HUONE Clarke Quay, 3D River Valley Road, #03-01, Clarke Quay, Singapore 179023'
+
+    # extract capacity
+    cap_html = indiv_venue.find('p', class_='persons-count')
+    capacity = ''
+    if cap_html:
+        capacity = cap_html.text.split(' ')[3].strip()
+
+    # extract description
+    desc_html = indiv_venue.find_all('div', class_='lead')[0]
+    desc = ''
+    if desc_html:
+        desc = desc_html.text.strip()
+    
+    # extract image links
+    venue_images = str(indiv_venue.find('div', class_='image-lift three-images'))
+    images = extract_huone_image_links(venue_images)
+
+    # extract facilities
+    facs = indiv_venue.find('div', class_='amenities block')
+    facilities = []
+    for fac in facs.find_all('li'):
+        facilities.append(fac.text.strip())
+
+    # extract price
+    price_html = facs.find('p')
+    price = ''
+    if price_html:
+        price = FormatPrice(price_html.text.strip())
```

Comment:
The FormatPrice doesnt work this way hahaa It doesnt take in a string. It is assuming that the input is an object of the format {dayOfWeek:"", time:"",pricing:""} and the main functionality of the function is just to fix the dayOfWeek.

For your case, you can just replace line 53 with 

**price = {
"dayOfWeek": "Inquire for day availabilities",
"time": "Inquire for timing availabilities",
"pricing": FormatSentence(price_html.text.strip())
}**

FormatSentence has to be imported from DataFormatting folder. I notice that the HALF DAY etc are in caps so we should convert that to lower case.